src/test/java/com/epam/ld/module2/testing/template/TemplateEngineTest.java

```diff
@@ -1,4 +1,170 @@
 package com.epam.ld.module2.testing.template;
 
+import com.epam.ld.module2.testing.Client;
+import com.epam.ld.module2.testing.datasource.ConsoleInput;
+import com.epam.ld.module2.testing.datasource.FileInput;
+import com.epam.ld.module2.testing.datasource.Input;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 public class TemplateEngineTest {
+
+    @Test
+    public void checkGenerateMessageWithValidTemplateFromConsole() {

```

Comment:
Isn't very clear why are we creating `ConsoleInput` and then copying data from it.
Also, seems like classes `ConsoleInput` `FileInput` `Input` are not used in production code.